composer.json

```diff
@@ -20,8 +20,8 @@
 	},
 	"require-dev": {
 		"composer/ca-bundle": "1.5.7",
-		"squizlabs/php_codesniffer": "3.10.3",
-		"wp-coding-standards/wpcs": "~3.1.0",
+		"squizlabs/php_codesniffer": "3.13.2",
```

Comment:
Since squizlabs/PHP_CodeSniffer [has been abandoned](https://github.com/squizlabs/PHP_CodeSniffer/issues/3932) and replaced by [PHPCSStandards/PHP_CodeSniffer](https://github.com/PHPCSStandards/PHP_CodeSniffer), should we switch to using the new location at this time?

Reply:
The package name is actually unchanged, it's still `squizlabs/php_codesniffer`